scripts/aws/after_install-ec2-user.sh

```diff
@@ -1,41 +1,48 @@
 #!/bin/bash
+APP_ROOT=/var/www/squacapi
+dest=$APP_ROOT/releases/$DEPLOYMENT_GROUP_NAME/$DEPLOYMENT_ID
+mkdir -p $dest
 
-dest=/var/www/releases/$DEPLOYMENT_GROUP_NAME/$DEPLOYMENT_ID
-mv /var/www/releases/tmp $dest
-aws s3 cp s3://squacapi-config/bash/$DEPLOYMENT_GROUP_NAME.env  $dest/app/.env
+# Move app out of tmp and into destination
+mv $APP_ROOT/releases/tmp/* $dest
+
+# Copy environment file
+aws s3 cp s3://squacapi-config/bash/squacapi-$DEPLOYMENT_GROUP_NAME.env  $dest/app/.env
+source $dest/app/.env
 
 echo after s3 call
 echo `ls $dest/app/.env `
 
 # virtual env vars
 export WORKON_HOME=/var/.virtualenvs
 export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3
-cd $dest
 source /usr/local/bin/virtualenvwrapper.sh
-source $dest/app/.env
+export VIRTUALENVWRAPPER_ENV_BIN_DIR=bin
+
 echo dest=$dest
-export PATH=$PATH:/usr/local/bin
+cd $dest
 
 # delete virtualenv so packages are consistent
-rmvirtualenv $DEPLOYMENT_GROUP_NAME
-mkvirtualenv $DEPLOYMENT_GROUP_NAME 2>&1
-workon $DEPLOYMENT_GROUP_NAME 2>&1
+rmvirtualenv squacapi-$DEPLOYMENT_GROUP_NAME
+mkvirtualenv squacapi-$DEPLOYMENT_GROUP_NAME 2>&1
+workon squacapi-$DEPLOYMENT_GROUP_NAME 2>&1
 
 # if staging, bootstrap
-if [ $DEPLOYMENT_GROUP_NAME == 'staging-squacapi' ]; then
+if [ $DEPLOYMENT_GROUP_NAME == 'staging' ]; then
     python $dest/app/manage.py bootstrap_db --days=7
 fi
 
-# both staging and prod use production.txt
+# All groups use production.txt
 pip3 install  -r $dest/requirements/production.txt
 python $dest/app/manage.py migrate
 
 # if production, update cronjobs
-if [ $DEPLOYMENT_GROUP_NAME == 'squacapi' ]; then
+if [ $DEPLOYMENT_GROUP_NAME == 'production' ]; then
+    python $dest/app/manage.py crontab -r
```

Comment:
Did you test this? The documentation says to do `python manage.py crontab remove`

Reply:
Good catch, I had done it outside the vm and thats the normal crontab syntax.